packages/repository/test/acceptance/has-many.relation.acceptance.ts

```diff
@@ -59,6 +55,43 @@ describe('HasMany relation', () => {
     expect(persisted.toObject()).to.deepEqual(order.toObject());
   });
 
+  it('can patch many instances', async () => {
+    await createSamples();
+    await testPatch();
+
+    async function testPatch() {
+      const patchObject = {description: 'new order'};
+      const arePatched = await customerOrders.patch(patchObject);
+      // tslint:disable-next-line: no-unused-expression
+      expect(arePatched).to.equal(2);
+      const patchedItems = await customerOrders.find();
+      expect(patchedItems).to.have.length(2);
+      patchedItems.forEach(order => {
+        expect(order.description).to.eql('new order');
+        expect(order.customerId).to.eql(existingCustomerId);
+      });
+    }
+  });
+
+  it('can delete many instances', async () => {
+    await createSamples();
+    await testDelete();
+
+    async function testDelete() {
+      const deletedOrders = await customerOrders.delete();
+      // tslint:disable-next-line: no-unused-expression
+      expect(deletedOrders).to.equal(2);
+      const relatedOrders = await customerOrders.find();
+      // tslint:disable-next-line: no-unused-expression
+      expect(relatedOrders).to.be.empty;
+    }
+  });
+
+  async function createSamples() {
```

Comment:
This test helper makes tests difficult to understand because it's not clear what kind of sample data was created. At minimum, please rename it to `createTwoSamples()`.

To be honest, I think this helper function is a wrong abstraction. A better solution is to create a test-data builder as described in our [docs](http://loopback.io/doc/en/lb4/Testing-your-application.html#use-test-data-builders) and then call this builder multiple times in each test.

```ts
it('can patch many instances', async () => {
  await givenCustomerOrder({description: 'order 1'});
  await givenCustomerOrder({description: 'order 2'});
  // run the test
});
```